base/uk.ac.stfc.isis.ibex.ui.graphing/src/uk/ac/stfc/isis/ibex/ui/graphing/websocketview/MatplotlibFigure.java

```diff
@@ -66,8 +71,12 @@ public MatplotlibFigure(Composite parent, int style, String url, int figureNumbe
 		labelConnectionStatus.setLayoutData(new GridData(SWT.FILL, SWT.TOP, true, false));
 		labelConnectionStatus.setText(viewModel.getPlotName().getValue());
 		
+		plotMessage = new Label(container, SWT.NONE);
+		plotMessage.setLayoutData(new GridData(SWT.FILL, SWT.TOP, true, false));
+		plotMessage.setText("");
+		
 		plotCanvas = new Canvas(container, SWT.NONE);
-		plotCanvas.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
+		plotCanvas.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true, 2, 1));
```

Comment:
It's not clear to me what these numbers are doing and look a bit magic. Is this something that could be a named constant?